src/smc.jl

```diff
@@ -41,6 +41,7 @@ function AbstractMCMC.sample(
     particles = ParticleContainer(
         [Trace(model, TracedRNG()) for _ in 1:(sampler.nparticles)], TracedRNG()
     )
+    seed_from_rng!(particles, rng)
```

Comment:
Isn't it a bit easier to allow e.g. to pass `rng` to the `ParticleContainer` constructor?

Reply:
Yes, good point I'll add a constructor